### ISSUE TITLE
Changing the system file picker

### ISSUE THREAD

**Issue by marco-brandizi** [PhilippC/keepass2android#1333]
The first time I use "System File Picker" to open a local database, I'm asked to choose the file manager to use for that, if I choose the stock app for that, K2A opens the database correctly, included the case the .kdbx is in external SD card.

If instead I choose another app (File Manager+), K2A tells me the database is read-only and it becomes impossible to choose another system file picker. I had to uninstall/reinstall the initially selected app to fix this.

K2A should allow for changing the file picker.

**Comment by PhilippC** [comment]
I don't think this is possible (without importing the file to app storage). Please reopen if I missed something.

**Comment by marco-brandizi** [comment]
HI @PhilippC, I don't get your comment. I'm saying K2A should have some feature to change the app that is used as System File Picker. The first time I use it, I'm presented with such a choice (I can choose between the stock file manager and the 'File Manager+' app), after the first time I'm stuck with what I initially choose.

**Comment by PhilippC** [comment]
are you able to record a video of that or take some screenshots? Sounds like a behavior I haven't heard of  or seen before.
